components/02-elements/button/_button.scss

```diff
@@ -1,36 +1,37 @@
 $button-height                    : 48px !default;
 $button-width                     : 48px !default;
-$button-padding                   : 0 35px !default;
+$button-padding                   : 0 $padding-base !default;
 $button-margin                    : 0 0 5px 0 !default;
 $button-border                    : none !default;
+$button-border-radius             : 100px !default;
 $button-hover-border              : none !default;
 $button-background                : $color-primary !default;
-$button-hover-background          : darken($color-primary, 20%) !default;
+$button-hover-background          : $gray-dark !default;
 $button-outline                   : none !default;
-$button-font-family               : $font-family-sans-serif !default;
+$button-font-family               : $font-family-base-bold !default;
 $button-font-size                 : $font-size-base !default;
 $button-font-color                : $white !default;
-$button-hover-font-color          : $white !default;
+$button-hover-font-color          : $color-primary !default;
 
 // Secondary
-$button-secondary-border          : 3px solid $gray-light !default;
+$button-secondary-border          : 1px solid $color-primary !default;
 $button-secondary-hover-border    : $color-primary !default;
```

Comment:
it's hover border color... I know it's long but we should keep naming descriptive

Reply:
Updated